prog.go

```diff
@@ -565,37 +622,44 @@ var haveProgTestRun = internal.FeatureTest("BPF_PROG_TEST_RUN", "4.12", func() e
 	return err
 })
 
-func (p *Program) testRun(in []byte, repeat int, reset func()) (uint32, []byte, time.Duration, error) {
-	if uint(repeat) > math.MaxUint32 {
-		return 0, nil, 0, fmt.Errorf("repeat is too high")
+func (p *Program) testRun(opts *RunOptions) (uint32, time.Duration, error) {
+	if len(opts.Data) == 0 {
```

Comment:
data_in must be zero for `SK_LOOKUP`, `RAW_TRACEPOINT` and `RAW_TRACEPOINT_WRITABLE`, so let's not make the assumption that `Data` is/needs to be populated.

Reply:
Yeah I agree. I think there was some discussion above. This was a carry-over from existing behavior, so probably best to change in a separate PR. I can file a followup.